csrankings-h.csv

```diff
@@ -442,6 +442,7 @@ Heiko Mantel,TU Darmstadt,http://www.mais.informatik.tu-darmstadt.de/Heiko_Mante
 Heiko Neumann,University of Ulm,http://www.informatik.uni-ulm.de/ni/mitarbeiter/HNeumann/index.html,0NbVSbMAAAAJ
 Heiko Paulheim,University of Mannheim,https://www.uni-mannheim.de/dws/people/professors/prof-dr-heiko-paulheim,SkSl3NkAAAAJ
 Heiko Röglin,University of Bonn,http://www.roeglin.org,NLLMDCkAAAAJ
+Heiko Schwarz,Freie Universitaet Berlin,https://iphome.hhi.de/schwarz/,41YpebQAAAAJ

```

Comment:
Does not appear to be in CS. Please clarify.

Reply:
In October 2017, Heiko Schwarz became Professor at the FU Berlin. But it seems that he is also heading the research group "Image and Video Coding" at the Fraunhofer Heinrich Hertz Institute. Since I have not found anything official, that he appoints 75%+ of his time at FU Berlin, I removed him.

https://www.fu-berlin.de/informationen-fuer/beschaeftigte/personalia/neuberufen/neuberufene-aktuell/schwarz/index.html